projects/examples/src/components/quick-search/filters/quick-search-filters.example.component.ts

```diff
@@ -0,0 +1,174 @@
+/*!
```

Comment:
Did you create this using the generator created by Yumin? Or copy pasted from another example? Please be sure to always use the generator, we don't want remnants of other tests.

Reply:
Will use the generator in the future.